mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/mail/CassandraMessageWithAttachmentMapperTest.java

```diff
@@ -37,6 +38,7 @@
 
     @BeforeClass
     public static void setUpClass() {
```

Comment:
If I launch only this test class in my IDE, it will start cassandra twice?

Reply:
Probably. Better solutions?

Reply:
Use a rule that manage the singleton.
BTW, test ordering is not supposed to be stable, this solution is not ok IMO.